airflow/plugins/operators/amplitude_to_flattened_json.py

```diff
@@ -49,11 +63,30 @@ def amplitude_to_df(start: str, end: str, api_key: str = None, secret_key: str =
 
 class AmplitudeToFlattenedJSONOperator(BaseOperator):
     """
-    An operator that will download data from Amplitude and upload the resulting flattened JSON to GCS.
+    An operator that will download data from Amplitude and upload the
+    resulting flattened JSON to GCS.
     """
 
-    def __init__(self, **kwargs):
-        super().__init__(self, **kwargs)
+    def __init__(self, app_name, **kwargs):
+        self.app_name = app_name
+        super().__init__(**kwargs)
 
     def execute(self, context):
-        pass
+        start_datetime = context.get("execution_date").set(hour=0, minute=0, second=0)
+
+        start = start_datetime.strftime(DATE_FORMAT)
+        end = (start_datetime + timedelta(hours=23)).strftime(DATE_FORMAT)
+
+        events_df = amplitude_to_df(start, end, app_name=self.app_name)
+
+        events_jsonl = "\n".join(
+            [
+                json.dumps(event)
+                for event in json.loads(events_df.to_json(orient="records"))
+            ]
+        )
```

Comment:
Maybe confirm that Pandas really can't write out JSON lines?

Reply:
It seems like it should be able to - since you can read JSON lines directly? But I'm terrible with remembering how to use pandas 😅